corefxlab.sln

```diff
@@ -144,6 +144,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Text.Utf8String", "s
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Text.Utf8String.Tests", "tests\System.Text.Utf8String.Tests\System.Text.Utf8String.Tests.csproj", "{C6F061D6-18D6-41E3-A692-C0603A994F30}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Server", "samples\SimpleHttp\Server.csproj", "{5B4ACAF4-9ECB-431E-A37B-960A224A9B40}"
```

Comment:
If we are adding these projects to the solution, we should add
`<Import Project="..\..\tools\common.props" />`
to the csproj files to bring in the new compiler/etc.